src/ProxyEvent.ts

```diff
@@ -56,20 +57,27 @@ export interface ProxyEvent {
         accountId: string,
         apiId: string,
         httpMethod: string,
+        authorizer?: {
+            [name: string]: any,
+        },
```

Comment:
Usually when indexed (map) objects are written they're kept on a single line, ie: `authorizer?: {[name: string]: any),`.  The reason is it's *possible* for there to be other properties on the object that override the typing on the index but it's really unlikely.

Like I've seen something like this make sense maybe once:

```
authorizer: {
    [name: string]: any,
    specificThing: string
}
```

Also to keep with the nice alphabetical sorting this should go above `httpMethod`.  Same goes for `RouterEvent`.